tests/integration_tests/modules/test_ansible.py

```diff
@@ -8,6 +10,20 @@
 # This instrumentation allows the test to run self-contained
 # without network access or external git repos.
 
+proxy_name = 'squid.internal:3128'
+try:
+    socket.gethostbyname(proxy_name)
+    if os.environ["PATH"].startswith("/jenkins"):
```

Comment:
Open to ideas if someone has a better suggestion than checking if squid.internal is resolvable and checking the `/jenkins` path. This seems to work locally, but there may be better ways.

Reply:
What about checking if some Jenkins' environment variable is set, e.g., `JENKINS_HOME` or `JOB_NAME`?

Reply:
It looks great, thank you.